config/locales/odp.yml

```diff
@@ -24,7 +24,7 @@ en:
       - ['approval', 'Planning approval']
       - ['approval.reservedMatters', 'Approval of reserved matters']
       - ['complianceConfirmation', 'Written confirmation of compliance with a planning condition']
-      - ['environmentalImpact', 'Environmental Impact Decision']
+      - ['environmnentalImpact', 'Environmental Impact Decision']
```

Comment:
```suggestion
      - ['environmentalImpact', 'Environmental Impact Decision']
```

Reply:
I've basically copy and paste, I saw them as well